### ISSUE TITLE
Misprint in readme:showLeaderboard

### ISSUE THREAD

**Issue by rotoxl** [ptgamr/cordova-google-play-game#14]
In readme "Show specific leaderboard" theres a misprint:

```javascript
var data = {
    leaderboardId: "board1"
};
googleplaygame.showLeaderboard(data);
```

(says ```googleplaygame.showLeaderboard(leaderboardId);```)

**Comment by Samleo8** [comment]
Yes, this desperately needs to be changed.. Caused me much problems. 
README also needs to be edited for the cordova-plugin-play-game-services plugin by @artberri which forks this one.
